fenix/app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -252,12 +252,12 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
                 components.core.engine.warmUp()
             }
 
-            // We need to always initialize Glean and do it early here.
-            initializeGlean()
-
             // Attention: Do not invoke any code from a-s in this scope.
             val megazordSetup = finishSetupMegazord()
 
+            // We need to always initialize Glean and do it early here.
+            initializeGlean()
+
```

Comment:
Can this change be safely reverted now?

Reply:
Yes, right! I missed that. It will be dealt most likely in this [PR](https://github.com/mozilla-mobile/firefox-android/pull/3104)